cli/cmd/predict.go

```diff
@@ -103,7 +102,7 @@ var predictCmd = &cobra.Command{
 			for _, prediction := range predictResponse.ClassificationPredictions {
 				if prediction.PredictedClassReversed != nil {
 					json, _ := json.Marshal(prediction.PredictedClassReversed)
-					fmt.Println(s.TrimPrefixAndSuffix(string(json), "\""))
+					fmt.Println(libstrings.TrimPrefixAndSuffix(string(json), "\""))
```

Comment:
nit: Can you use `s.*` for now since that's what we use elsewhere? We can discuss if we want to change to `libstrings`, and if so, we should change them all at once.